aws/resource_aws_globalaccelerator_accelerator.go

```diff
@@ -13,6 +13,10 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
 )
 
+// Global Route53 Zone ID for Global Accelerators, exported as a
+// convenience attribute for Route53 aliases.
+const globalAcceleratorRoute53ZoneID = "Z2BJ6XQ5FK7U4H"
```

Comment:
Obtained from http://docs.aws.amazon.com/Route53/latest/APIReference/CreateAliasRRSAPI.html, followed the pattern used by `aws_cloudfront_distribution` which also has a global Route53 zone id.

Reply:
That link didn't work for me, but the zone id is also mentioned on https://docs.aws.amazon.com/Route53/latest/APIReference/API_AliasTarget.html

Reply:
That's the one, thanks!